design/comments.go

```diff
@@ -65,12 +65,29 @@ var identityRelationData = a.Type("IdentityRelationData", func() {
 	a.Required("type")
 })
 
+var commentRelationshipsArray = a.MediaType("application/vnd.comment-relationships+json", func() {
```

Comment:
Could we not use the JSON API Helper functions here? 
https://github.com/almighty/almighty-core/blob/master/design/jsonapi_helpers.go#L99

Usage https://github.com/almighty/almighty-core/blob/master/design/spaces.go#L45